### PR TITLE
refactor(site): heading sliders changed, all texts has been changed

### PR DESCRIPTION
**Pull request by akhnos**

### CODE REVIEW

File: docs/site/src/app/pages/home/home.component.html [hunk 1]
```diff
@@ -3,54 +3,62 @@
...
       </h1>
-      <div
-        fxLayout="row"
-        fxLayoutAlign="center center"
-        fxLayout.lt-sm="column"
-        class="button-group"
-      >
-        <!-- <a mat-icon-button href="https://twitter.com/SpicaEngine" target="_blank">
+      <p>
+        It is the fastest way to create a flexible backend for you next web or mobile application.
```
Comment (TolgaYigit) [inline docs/site/src/app/pages/home/home.component.html:23]:
```suggestion
        It is the fastest way to create a flexible backend for your next web or mobile application.
```

File: docs/site/src/app/pages/home/home.component.html [hunk 2]
```diff
@@ -3,54 +3,62 @@
...
+      fxLayout="column"
+      fxLayoutAlign="flex-start"
+      class="button-group"
+      intersect
+      #intersect13="intersect"
+      [@flyOneByOne]="intersect13.isIntersecting"
+    >
       <h1>
-        The first open source <br />
+        The open source <br />
```
Comment (TolgaYigit) [inline docs/site/src/app/pages/home/home.component.html:19]:
```suggestion
        An open source <br />
```

File: docs/site/src/app/pages/home/home.component.html [hunk 3]
```diff
@@ -3,54 +3,62 @@
...
-      <div
-        fxLayout="row"
-        fxLayoutAlign="center center"
-        fxLayout.lt-sm="column"
-        class="button-group"
-      >
-        <!-- <a mat-icon-button href="https://twitter.com/SpicaEngine" target="_blank">
+      <p>
+        It is the fastest way to create a flexible backend for you next web or mobile application.
+        Decrease lift off time with Managed Spica Services.
```
Comment (TolgaYigit) [inline docs/site/src/app/pages/home/home.component.html:24]:
```suggestion
        Decrease lift-off time with Managed Spica Services.
```

File: docs/site/src/app/pages/home/home.component.html [hunk 4]
```diff
@@ -113,10 +120,10 @@ <h2>Data Model Customization</h2>
   </div>
   <div fxFlex="33" fxFlex.lt-md="100">
     <img src="assets/artboard/composer.svg" alt="Composer" />
-    <h2>Frontend Composer</h2>
+    <h2>Developer Focused</h2>
     <p>
-      Why should you need another branch for frontend if you know what you want to show? With the
-      component-based interface composer, you can make UIs without coding.
+      Auto generated APIs will decrease your lift off times for your projects. Everything in Spica
```
Comment (TolgaYigit) [inline docs/site/src/app/pages/home/home.component.html:125]:
```suggestion
      Auto-generated APIs will decrease your lift-off times for your projects. Everything in Spica
```

File: docs/site/src/app/pages/home/home.component.html [hunk 5]
```diff
@@ -113,10 +120,10 @@ <h2>Data Model Customization</h2>
...
+      designed for developers to create custom and flexible backend.
```
Comment (TolgaYigit) [inline docs/site/src/app/pages/home/home.component.html:126]:
```suggestion
      designed for developers to create a custom and flexible backend.
```

**Review by htuna07 — APPROVED** [review]

**Review by TolgaYigit — COMMENTED** [review]
I have found some typos in the content. Can you please check them?

**Review by TolgaYigit — APPROVED** [review]

**Comment by htuna07** [comment]
Can we update this title with a more descriptive one, please?

**Pull request merged**